ceph/datadog_checks/ceph/ceph.py

```diff
@@ -188,7 +203,13 @@ def _extract_metrics(self, raw, tags):
             self.log.debug('Error retrieving osd_pool_stats metrics')
 
         try:
-            osdstatus = raw['status']['osdmap']['osdmap']
+            raw_osdstatus = raw['status']['osdmap']
+            if 'osdmap' in raw_osdstatus:
+                osdstatus = raw_osdstatus['osdmap']
+            else:
+                osdstatus = raw_osdstatus
```

Comment:
When would `raw['status']['osdmap']` be a value vs. a dictionary?

Reply:
octopus vs non octopus versions have different formats. it's part of the changes in octopus, and the original issue that brought this up. See issue #8067 for more info on this format.